deployment/hasura/metadata/databases/tables/sequencing/parcel.yaml

```
@@ -0,0 +1,60 @@
+table:
+  name: parcel
+  schema: sequencing
+configuration:
+  custom_name: "parcel"
+object_relationships:
+  - name: channel_dictionary
+    using:
+      foreign_key_constraint_on: channel_dictionary_id
+  - name: command_dictionary
+    using:
+      foreign_key_constraint_on: command_dictionary_id
+  - name: sequence_adaptation
+    using:
+      foreign_key_constraint_on: sequence_adaptation_id
+array_relationships:
+  - name: parameter_dictionary
+    using:
+      foreign_key_constraint_on:
+        column: parcel_id
+        table:
+          name: parcel_to_parameter_dictionary
+          schema: sequencing
+  - name: expansion_sets
+    using:
+      foreign_key_constraint_on:
+        column: parcel_id
+        table:
+          name: expansion_set
+          schema: sequencing
+insert_permissions:
+  - role: aerie_admin
+    permission:
+      columns: "*"
+      check: {}
```

Minor: `set` field is missing. It goes after `check`

```suggestion
      check: {}
      set: 
          owner: "x-hasura-user-id"
```

---

I updated the `insert_permissions:` to have the `set`